src/main/resources/swagger.api/schemas/resourceEvent.json

```diff
@@ -11,6 +11,10 @@
       "description": "Resource event operation type",
       "$ref": "resourceEventType.json"
     },
+    "subType": {
```

Comment:
based on the field name, ```subType```, I would assume that this field is related to any resource and event type. However, in the description we are telling that this is for ```delete operation type```.

I might consider to change the description to something like ```Resource event operation sub type``` or field name to ```deleteEventSubType```. Then, similarly the corresponding changes for the schema: if we want to leave a possibility to use this field also for other event types in the future then I would change description in ```resourceEventSubType.json``` to ```Resource event operation sub type``` or the schema's name to ```resourceDeleteEventSubType``` (in case if it is just for delete event)